pages/thank-you.js

```diff
@@ -46,16 +45,16 @@ export default function Home() {
                   ". We're looking for volunteer contributors to help make it better. Report bugs or propose ideas for improvements by submitting GitHub issues, or even submit little pull requests as proposals."
                 }
               </p>
-              <h3 className="m-5 text-xl font-bold">{'Reach Out to Us'}</h3>
+              <h3 className="m-0 mt-10 mb-2 text-3xl font-bold">{'Reach Out to Us'}</h3>
```

Comment:
Please put a newline before the `{` and after the `}`
Alternatively run `eslint . --fix` if you have eslint installed.